src/taskManager/UserInterface.java

```diff
@@ -0,0 +1,114 @@
+package taskManager;
+
+import java.util.Scanner;
+
+public class UserInterface {
+    Scanner sc = new Scanner(System.in);
+    TaskList taskList = new TaskList();
+
+    public void startUI(){
+        while(true){
+            printCommandOptions();
+        }
+    }
+
+    public void printCommandOptions() {
+        System.out.println("Select command you would like to implement from following:");
+        System.out.println("1 - add Task");
+        System.out.println("2 - edit Task");
+        System.out.println("3 - remove Task");
+        System.out.println("4 - print Tasks List");
+        System.out.println("5 - change task status");
+        System.out.println("exit - exit the application");
+        handleInput();
+    }
+
+    public void handleInput() {
+        String command = sc.nextLine();
+        switch (command) {
```

Comment:
An alternative to this switch statement could be fetching command handlers from a map.
Constructor could register a separate handler for each command type in this case and `handleInput` would just get handler from the map and execute it.

Reply:
It sounds very 'senior-level.' I like it. 😊

Reply:
done in a very simple way